resources/innosetup/setup.iss

```diff
@@ -61,25 +61,28 @@ Root: HKLM; Subkey: "SYSTEM\CurrentControlSet\Services\WebClient\Parameters"; Va
 ; Remove legacy AutoStart entry; see: https://github.com/cryptomator/cryptomator-win/issues/33
 Root: HKCU; Subkey: "SOFTWARE\Microsoft\Windows\CurrentVersion\Run\"; ValueType: none; ValueName: "Cryptomator"; Flags: dontcreatekey deletevalue
 
+; Remove legacy ProgID introduced in 1.5.12-beta1
+Root: HKCR; Subkey: "EncryptedCryptomatorVaultFile"; Flags: dontcreatekey deletekey
+
```

Comment:
Would it harm to keep this value? If not, don't bother removing it, as we should not pile up cleanup code for pre-release versions.

Reply:
It will get removed anyway during uninstall of the beta (note the `uninsdeletekey`):

https://github.com/cryptomator/cryptomator-win/blob/adc0e9139e98ef87441401603f79aab4981845ff/resources/innosetup/setup.iss#L65

Reply:
> Would it harm to keep this value? If not, don't bother removing it, as we should not pile up cleanup code for pre-release versions.

No, but it collides with Microsoft's specifications and could confuse users if the decide to snoop around in the registry. From a technical perspective, it doesn't matter.

> It will get removed anyway during uninstall of the beta (note the uninsdeletekey):

Correct, but since we don't call the uninstaller on update, this is far from a guarantee. 

Resolved by 7d3cb888975114ef02ccd2822780aabe0294dee0

Reply:
> we don't call the uninstaller on update

But InnoSetup will append the install log, so it will get removed during uninstall eventually.